code/game/gamemodes/cult/cult_items.dm

```diff
@@ -596,7 +596,23 @@
 				L.visible_message("<span class='warning'>[src] bounces off of [L], as if repelled by an unseen force!</span>")
 		else if(!..())
 			if(!L.null_rod_check())
-				L.Weaken(6 SECONDS)
+				var/datum/status_effect/cult_stun_mark/S = L.has_status_effect(STATUS_EFFECT_CULT_STUN)
+				if(S)
```

Comment:
Same here

Reply:
Sorry for the late response, I somehow managed to miss this comment. But it looks good as is for now